src/index.js

```diff
@@ -736,7 +725,8 @@ class JanusAdapter {
           this.publisher.unreliableChannel.send(JSON.stringify({ clientId, dataType, data }));
           break;
         default:
-          console.err("JanusAdapter.unreliableTransport must be one of ['websocket', 'datachannel'].");
```

Comment:
Obviously not important now that they're gone, but I think these `console.err` calls were meant to be `console.error`.   I was running a version of `naf-janus-adapter` from before this commit when `console.err` was undefined.
![image](https://user-images.githubusercontent.com/4072106/46381469-4c589f00-c65b-11e8-8822-517a0e252167.png)